crates/bevy_ui/src/lib.rs

```diff
@@ -161,7 +161,9 @@ impl Plugin for UiPlugin {
         #[cfg(feature = "bevy_text")]
         app.add_plugins(accessibility::AccessibilityPlugin);
         app.add_systems(PostUpdate, {
+            #[allow(clippy::let_and_return)]
```

Comment:
No longer needed.

Reply:
```suggestion
```

Reply:
Fixed in 0ec58548217ecc1d4e99bb9c7223988bf51ea1a5. I had to create another system set since there were two ambiguity modifiers. (I didn't realize a system could be in more than one set. Cool!)